fcrepo-configs/src/main/resources/config/file-s3/repository.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name" : "repo",
+    "jndiName" : "",
+    "workspaces" : {
+        "predefined" : ["default"],
+        "default" : "default",
+        "allowCreation" : true,
+        "cacheSize" : 10000
+    },
+    "storage" : {
+        "persistence": {
+            "type": "file",
+            "path" : "${fcrepo.object.directory:target/objects}"
+        },
+        "binaryStorage" : {
+            "type" : "s3",
+            "username" : "${aws.accessKeyId}",
```

Comment:
Since these are json files, I am not sure how to add documentation inside of them without requiring client-side pre-processing. But improving the wiki documentation would certainly be helpful.

As a start, we could update the following:
- https://wiki.duraspace.org/display/FEDORA4x/Best+Practices+-+Fedora+Configuration
- https://wiki.duraspace.org/display/FEDORA4x/Configuring+JDBC+Object+Store
- and a new AWS/S3 page